### ISSUE TITLE
Neurogenic Bowel - not found on search.   Anganglionic megacolong terms and neurogenic bladder are present

### ISSUE THREAD

**Issue by TonyRoscioli** [obophenotype/human-phenotype-ontology#7463]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Hi @TonyRoscioli  -- as far as I know, neurogenic bowel is a description of the pathogenesis and in any individual patient one would have observable features such as diarrhea or constipation rather than a clinical picture that on its own would lead to the diagnosis of neurogenic bowel. It may also be best to retire the term neurogenic bladder, which seems to be more commonly used in the genetic literature so HPO has many annotations to it.
Thoughts?

**Comment by pnrobinson** [comment]
I will close this for now, we can reopen if needed.
